.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
```

Comment:
https://github.com/pre-commit/pre-commit-hooks/releases/tag/v2.3.0 is from 2019. why are you suggesting such an old version?